client/src/components/Tabs/README.md

```diff
@@ -33,3 +33,86 @@ clickable tab.
  * `tabClassName` (string): Class to use for the tab.
 
  _NOTE:_ For other properties, please refer to the [reactstrap Tabs](https://reactstrap.github.io/components/tabs/) documentation.
+
+# Interacting with Tabs from a child component
+
+Components inside a tab can retrieve information about their parent using these method:
```

Comment:
`injectTabContext` is an HOC. It could be used with a functional component as well. Not sure there's much point dividing up a list of three items.

Reply:
I think there is, basically

If you have a functional component then do this:
...

If you have a class component then do this:
...

Doesn't seem like much extra work to make the instructions better?

Reply:
It's not clear to me this makes the doc better. You could argue that `injectTabContext` and `useTabContext` should be together  because they essentially do the same thing and `useTabFirstShow` should be on its own.

At the of of the day, these 3 methods do related things. I think it makes sense to discuss them as a whole.